### ISSUE TITLE
Help Wanted

### ISSUE THREAD

**Issue by anuradhawick** [GATB/gatb-core#39]
Hi,

I am interested in using GATB in long reads analysis as they seem more appealing with HiFi reads and ultra-long ONT reads. Is GATB specifically for NGS short reads? Will there be future support and updates for long reads data? This is to ensure that I am using the right technologies for the right purpose. 

Comments will be highly appreciated.

Thank you!

**Comment by rchikhi** [comment]
yes it's mainly for doing k-mer tasks, with not too long k (although sizes of 1000s are supported, the toolkit would be rather slow), so the main usage was indeed short reads. However if you'd like to manipulate de Bruijn graphs or kmers on HiFi data, then GATB can help.
